spec/spec_helper.rb

```diff
@@ -3,6 +3,9 @@
 require File.expand_path("../../config/environment", __FILE__)
 require 'rspec/rails'
 require 'capybara/rails'
+require 'capybara/email/rspec'
+require 'sidekiq/testing'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.